data-prepper-plugins/elasticsearch/odfe_security.md

```diff
@@ -0,0 +1,19 @@
+## Opendistro For Elasticsearch
+
+
+The Elasticsearch sink can send trace data to opendistro-for-elasticsearch (ODFE) cluster using user credential assigned with a role that has the below required permissions. 
+
+### Cluster permissions
+
+- `cluster_all`
+- `indices:admin/template/get`
+- `indices:admin/template/put`
+
+Note that `indices:admin/template/*` need to be in cluster permissions.
+
+### Index permissions
+
+- `Index`: `otel-v1*`; `Index permissions`: `indices_all`
+- `Index`: `.opendistro-ism-config`; `Index permissions`: `indices_all`
+
```

Comment:
Could you add the following. We still need manage alias permission for _alias API

`Index`: `*`; `Index permission`: `manage_aliases`

It is already in the master.